packages/calypso-e2e/src/lib/components/support-component.ts

```diff
@@ -38,116 +14,79 @@ export class SupportComponent {
 	 */
 	constructor( page: Page ) {
 		this.page = page;
-	}
-
-	/**
-	 *
-	 */
-	async waitForQueryComplete(): Promise< void > {
-		await Promise.all( [
-			this.page.waitForSelector( selectors.resultsPlaceholderHelpCenter, { state: 'hidden' } ),
-			this.page.waitForSelector( selectors.resultsPlaceholder, { state: 'hidden' } ),
-		] );
+		this.helpCenter = this.page.locator( '.components-card__body > .inline-help__search' );
```

Comment:
Changed: locator now points to the results field (in red) not the entire Inline Help modal.

This is because there is no selector to select the entire modal that is visible.

<img width="391" alt="Screenshot 2023-05-17 at 3 48 52 PM" src="https://github.com/Automattic/wp-calypso/assets/6549265/4b1fb0af-17a8-4517-a9d0-de3654d03720">